website/docs/guides/getting-started.html.markdown

```diff
@@ -12,6 +12,9 @@ description: |-
 
 ## Kubernetes
 
+-> Visit the [Manage Kubernetes Resources via Terraform](https://learn.hashicorp.com/tutorials/terraform/kubernetes-provider?in=terraform/kubernetes&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE) Learn tutorial for an updated getting
```

Comment:
Should we remove the source tracking from the URL and just have "https://learn.hashicorp.com/tutorials/terraform/kubernetes-provider?in=terraform/kubernetes" ? It's probably misleading to analytics

Reply:
Regarding the language, the current phrasing sounds a bit like this guide is an "outdated" one and the Learn tutorial is the updated one. Is the intention to eventually replace this guide with Learn?

Reply:
Yes, I agree with @aareet about the word "updated."  Perhaps "interactive" would be a better description?